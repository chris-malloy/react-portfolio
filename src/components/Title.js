import React from 'react';

const Title = (props) => {
  return ( 
    <div id={props.id}>
      <a>
        <h5 className="center title">{props.title}</h5>
      </a>
    </div>
  )
}

export default Title;