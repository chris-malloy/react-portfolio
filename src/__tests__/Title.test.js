import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Title from '../components/Title';

configure({ adapter: new Adapter() });

describe('Title', () => {
  const title = shallow(<Title />);
  
  it('renders correctly', () => {
    expect(title).toMatchSnapshot();
  })
})