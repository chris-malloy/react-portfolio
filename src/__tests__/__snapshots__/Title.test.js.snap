// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Title />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <a>
        <h5
                className="center title"
        />
</a>,
      "id": undefined,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h5
          className="center title"
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "center title",
        },
        "ref": null,
        "rendered": null,
        "type": "h5",
      },
      "type": "a",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <a>
          <h5
                    className="center title"
          />
</a>,
        "id": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h5
            className="center title"
/>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "center title",
          },
          "ref": null,
          "rendered": null,
          "type": "h5",
        },
        "type": "a",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
